Python/initconfig.c

```diff
@@ -1642,6 +1651,21 @@ config_read_env_vars(PyConfig *config)
         config->safe_path = 1;
     }
 
+    const char *gil = config_get_env(config, "PYTHON_GIL");
+    if (gil != NULL) {
+#ifdef Py_GIL_DISABLED
+        if (strcmp(gil, "0") == 0) {
+            config->enable_gil = _PyConfig_GIL_DISABLE;
+        } else if (strcmp(gil, "1") == 0) {
+            config->enable_gil = _PyConfig_GIL_ENABLE;
+        } else {
+            return _PyStatus_ERR("PYTHON_GIL must be \"0\" or \"1\"");
+        }
```

Comment:
Format with `else if` and `else` on new lines (see [PEP 7](https://peps.python.org/pep-0007/#:~:text=Braces%20should%20be%20formatted%20as%20shown%3A))